package.json

```
@@ -86,7 +86,7 @@
     "@bbc/psammead-media-indicator": "6.1.0",
     "@bbc/psammead-media-player": "5.1.0",
     "@bbc/psammead-most-read": "6.0.28",
-    "@bbc/psammead-navigation": "9.1.1",
+    "@bbc/psammead-navigation": "9.1.2",
```

```suggestion
    "@bbc/psammead-navigation": "9.2.0",
```
I think this should be a minor package bump as component is changing behaviour (not just an internal bug fix) see https://semver.org/

---

Great suggestion @andrewscfc. Thanks for the review